pkg/k8sutil/cluster.go

```diff
@@ -34,7 +48,8 @@ func NewCluster(client *kubernetes.Clientset, expectedNodes int) (*Cluster, erro
 	return &Cluster{KubeClient: client, ExpectedNodes: expectedNodes}, nil
 }
 
-func (cl *Cluster) Health() ([]v1.ComponentStatus, error) {
+// ComponentsStatus returns the status of Lokomotive cluster components.
```

Comment:
This function is generic to all Kubernetes clusters, it's not specific to Lokomotive.
```suggestion
// ComponentsStatus returns the status of Kubernetes cluster components.
```